src/pages/user_profile/UserProfile.jsx

```diff
@@ -163,6 +163,9 @@ export default function UserProfile(props) {
         Full Name
       </h1>
       <div style={{ display: 'flex', alignItems: 'center' }}>
+        <label htmlFor="profileName" style={{ marginRight: 8 }}>
+          Profile name
+        </label>
```

Comment:
These labels add visual content to the screen that we don't want. Note the difference between dev on the left and this PR on the right. 

<img width="1499" height="521" alt="Image" src="https://github.com/user-attachments/assets/68f93fbd-249e-4a3c-8059-a919a22fb4cd" />